tests/shutdown/shutdown.pm

```diff
@@ -14,20 +14,25 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
-# G-Summary: support for saving and loading of hdd image
-# G-Maintainer: Vladimir Nadvornik <nadvornik@suse.cz>
+# Summary: support for saving and loading of hdd image
+# Maintainer: Vladimir Nadvornik <nadvornik@suse.cz>
 
 use strict;
 use base 'basetest';
 use testapi;
+use utils 'sle_version_at_least';
 
 sub run {
 
     select_console('root-console');
     if (get_var('DROP_PERSISTENT_NET_RULES')) {
         type_string "rm -f /etc/udev/rules.d/70-persistent-net.rules\n";
     }
-
+    unless (sle_version_at_least('12-SP1') && !check_var('VIRTIO_CONSOLE', 1)) {
```

Comment:
that `unless` paired with `!check_var` is really hard to follow. Can you use if?